02/PrintRecords.cpp

```diff
@@ -7,24 +7,95 @@ using namespace std;
 
 namespace samples
 {
+	void InputRecords(Record* info)
```

Comment:
it reads only one record.. so it should be InputRecord

also let's make it return.. then u don't need pass by reference in this function

Record InputRecord()